.github/workflows/composites/find-all-tests/action.yaml

```diff
@@ -0,0 +1,20 @@
+name: find all tests
```

Comment:
Maybe its best to rename this now as its purpose is to compile the code and find all tests?  I image by combining these two steps we save sometime?